x/mongo/driver/crypt.go

```diff
@@ -72,6 +73,7 @@ type crypt struct {
 	collInfoFn CollectionInfoFn
 	keyFn      KeyRetrieverFn
 	markFn     MarkCommandFn
+	tlsCfg     map[string]tls.Config
```

Comment:
Optional: Consider changing to `tlsConfig` to match the field name on `AutoEncryptionOptions` and `ClientEncryptionOptions`.

Reply:
Done!